src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -1,6 +1,7 @@
 /****************************************************************************
  *
  *   Copyright (c) 2013-2019 PX4 Development Team. All rights reserved.
+ *   Copyright (c) 2020-2021 ThunderFly s.r.o., All rights reserved,
```

Comment:
I am not sure it is the right thing to do to add `ThunderFly` as a copyright holder of this file.